sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -265,6 +339,28 @@ void gnrc_ipv6_netif_remove(kernel_pid_t pid);
 gnrc_ipv6_netif_t *gnrc_ipv6_netif_get(kernel_pid_t pid);
 
 /**
+ * @brief   Forward definition for gnrc_ndp_router_set_router()
+ *
+ * @note    Can't include net/gnrc/ndp/router.h since it includes
+ *          net/gnrc/ipv6/netif.h (This file.)
+ *
+ * @param[in] netif     The interface.
+ * @param[in] enable    Status for the GNRC_IPV6_NETIF_FLAGS_ROUTER flag.
+ */
+extern void gnrc_ndp_router_set_router(gnrc_ipv6_netif_t *netif, bool enable);
```

Comment:
Because they are dependent on the router discovery. 6LoWPAN has it slightly different e.g.